### ISSUE TITLE
NET Core3.1 test in macOS runner

### ISSUE THREAD

**Issue by muno92** [muno92/resharper_inspectcode#99]
Install .NET Core 3.1 (by setup dotnet action) is not work.

Because, .NET5 is already installed on Mac OS Runner's ~/.dotnet/dotnet  
https://github.com/muno92/resharper_inspectcode/runs/3231696660?check_suite_focus=true

**Comment by muno92** [comment]
- setup-dotnet Action is correct
  - Mac OS Runner is already installed mulitiple version .NET SDK.
    ```text
    $ dotnet --list-sdks
    2.1.302 [/Users/runner/.dotnet/sdk]
    2.1.403 [/Users/runner/.dotnet/sdk]
    2.1.507 [/Users/runner/.dotnet/sdk]
    2.1.607 [/Users/runner/.dotnet/sdk]
    2.1.701 [/Users/runner/.dotnet/sdk]
    2.1.818 [/Users/runner/.dotnet/sdk]
    3.1.101 [/Users/runner/.dotnet/sdk]
    3.1.201 [/Users/runner/.dotnet/sdk]
    3.1.302 [/Users/runner/.dotnet/sdk]
    3.1.416 [/Users/runner/.dotnet/sdk]
    5.0.102 [/Users/runner/.dotnet/sdk]
    5.0.202 [/Users/runner/.dotnet/sdk]
    5.0.302 [/Users/runner/.dotnet/sdk]
    5.0.404 [/Users/runner/.dotnet/sdk]
    ```
  - For build, it's not problem, but before version 2021.1.5, ReSharper CLI installation will be failed.
- Work around (may be)
  - Run dotnet tool command with --framework
    https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-tool-install
  - This issue is not necessary fixed (only test problem).

**Comment by muno92** [comment]
Resolved by #440
